src/lib/mocha/mocha-wrapper.js

```diff
@@ -10,41 +9,57 @@ var Mocha = require('mocha'),
 import {parseBoolean, parseNullableString, parseString} from '../environment-variable-parsers';
 import escapeRegExp from '../utils/escape-reg-exp';
 
-  var mochaConfig = JSON.parse(process.env.mochaConfig);
-mochaConfig.ui = 'fiberized-bdd-ui';
+class MochaWrapper {
+  constructor() {
```

Comment:
everything inside this constructor was just copy/pasted from the contents of the old file, nothing's been changed but adding the section for `process.env['chimp.files']`